partcad/src/partcad/project_factory_git.py

```
@@ -86,82 +113,113 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
+                                before = repo.active_branch.commit
+                                origin = repo.remote("origin")
+                                # Need to check for updates
+                                origin.fetch()
+                                repo.git.checkout(self.import_revision, force=True)
+                                origin.pull(force=True, rebase=True)
+                                os.utime(guard_path, (now, now))
+                            else:
+                                # No update was performed
+                                before = None
+
+                        if not before is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)